manuscript.Rmd

```diff
@@ -424,30 +457,73 @@ In that case, the readers can be certain, that they have received a complete acc
+
+Focusing on uncertainty reduction also explains two common practices of preregistration that do not align with a confirmatory research agenda.
+First, researchers seldomly predict precise numerical outcomes, instead they use preregistrations to describe the process that generates the results.
+Precise predictions would have very high theoretical risk (they are likely incorrect if the theory is wrong).
+A statistical procedure, may have high or low theoretical risk depending on the specifics of the model used.
+Specifying the process, therefore, is in line with the rational we propose here, but is incompatible with strictly confirmatory research.
+
+Second, researchers often have to deviate from the preregistration and make data-dependent decisions outside of the preregistration.
+If the only goal of preregistration is to ensure confirmatory research, such changes are not justifiable.
+However, under our rational some changes may be justified.
+Any change would increase the uncertainty about the theoretical risk and may even decrease the theorethical risk.
+The changes still may be worthwhile, if the negative outcomes may be offset by an increase in detectability due to the change.
+Consider a preregistration that failed to specify what to do with missing values where the researchers subsequently encounter missings.
+In such case, the detectability is zero, since the data cannot be analyzed without a decision about how to handle the missing data.
+Any such decision would constitute a deviation from the preregistration.
+Furthermore, a reader cannot rule out that the researchers leveraged the decision to decrease theoretical risk, i.e., picking among all options the one that delivers the most beneficial results for the theory.
+Whatever decision they make, increased uncertainty about the theoretical risk is inevitable and the expected epistemic value is decreased compared to a world where they anticipated the need to deal with missing data.
+However, it is still justified to deviate.
+After all they have not anticipated the case and are left with a detectablilty of zero.
+Any decision will increase detectability to a non-zero value, likely offsetting the increase in uncertainty.
+The researchers also may do their best to argue that the deviation was not motivated by increasing theoretical risk, thereby, decreasing the uncertainty.
+Maybe there is one decision that fits well with the theory or with the study design.
+Or, if there is no obvious candidate, the researchers could conduct a multiverse analysis of the available options to deal with missings to show the influence of the decision [@steegenIncreasingTransparencyMultiverse2016].
+
+Using reduction in uncertainty as the objective for preregistration does not only explain some existing behavior, it allows to form recommendations to improve the practice of preregistration.
```

Comment:
I don't understand "to explain some exisiting behavior".. of whom? and what exactly?